src/components/reviewMatches/MatchDetails.tsx

```diff
@@ -377,14 +377,18 @@ const MatchDetails = () => {
           <DialogContentText>{dialog.text}</DialogContentText>
         </DialogContent>
         <DialogActions>
-          <Button onClick={handleCancel}>Cancel</Button>
-          <Button onClick={handleConfirm} autoFocus>
-            {accept.isLoading || newGoldenRecord.isLoading ? (
-              <CircularProgress />
-            ) : (
-              'Confirm'
-            )}
-          </Button>
+          <CustomButton onClick={handleCancel} label="Cancel" />
+          <CustomButton
+            onClick={handleConfirm}
+            autoFocus
+            label={
```

Comment:
IsLoading should be added as a boolean prop.

Reply:
sure.